pkg/spec/resources_test.go

```diff
@@ -170,6 +171,85 @@ func TestFixSecrets(t *testing.T) {
 	}
 }
 
+func TestFixImageStreams(t *testing.T) {
+	appName := "testAppName"
+	tests := []struct {
+		name    string
+		input   []ImageStreamSpecMod
+		output  []ImageStreamSpecMod
+		success bool
+	}{
+		{
+			name: "passing one imageStream without name",
+			input: []ImageStreamSpecMod{
+				{},
+			},
+			output: []ImageStreamSpecMod{
+				{
+					ObjectMeta: meta_v1.ObjectMeta{
+						Name: appName,
+						Labels: map[string]string{
+							appLabelKey: appName,
+						},
+					},
+				},
+			},
+			success: true,
+		},
+		{
+			name: "passing one imageStream with name",
+			input: []ImageStreamSpecMod{
+				{
+					ObjectMeta: meta_v1.ObjectMeta{
+						Name: "routeName",
```

Comment:
Can we use name which is imageStream specific, might cause unnecessary confusion to new devs!